meteor/.eslintrc.js

```diff
@@ -68,10 +68,19 @@ module.exports = {
 	extends: [...commonExtends, 'plugin:react/recommended'],
 	plugins: [...commonPlugins, 'react'],
 	rules: {
-		'prettier/prettier': 'error',
+		'prettier/prettier': [
+			'error',
+			{
+				endOfLine: 'auto',
+			},
+		],
 	},
 	env: { es2017: true },
-	parserOptions: { sourceType: 'module', ecmaVersion: 2018 },
+	parserOptions: {
+		sourceType: 'module',
+		ecmaVersion: 2018,
+		tsconfigRootDir: __dirname,
+	},
```

Comment:
These changes i originally made while working in VSCode, which solved an error on the end of every line, in every `.ts` or `.tsx` file. The lower parts fixes an error at the imports, causing them to be unable to locate the tsconfig file.

As i am now using Webstorm too, i'd be fine with undoing these changes, if needed. However if they do not hurt anything, then they could possibly be helpful in the future, for anyone who'd chose to open the project in VSCode.

Reply:
We prefer to make our changes as IDE agnostic as possible. So it would be greatly apreciated that these changes would be reverted.

Sidenote: This sounds more like something you should change in your IDE than in the project configuration.

Reply:
Has been reverted.